Documentation/5.4.1/README.rst

```diff
@@ -1144,13 +1157,13 @@ where:
    * ``network-isolation.yaml`` Enables creation of networks for isolated overcloud traffic
    * ``net-bond-with-vlans.yaml`` Configures an IP address and a pair of bonded nics on each network
    * ``sriov-role.yaml`` Enables services required for Compute Sriov role
-   * ``neutron-sriov.yaml`` Neutron SRIOV specific parameter values
-   * ``avrs-role.yaml`` Enables services required for Compute Avrs role
-   * ``ironic-role.yaml`` Enables Ironic Inspector service for Controller role
    * **``ovs-hw-offload.yaml``** Enables OVS Hardware Offload on VRS Offload Compute nodes
```

Comment:
I don't think you require `` before and after ovs-hw-offload.yaml, host-config-and-reboot.yaml, mellanox-environment.yaml. Please see the rendered file to see exactly why I am saying this.